include/wx/generic/grid.h

```diff
@@ -2659,6 +2680,7 @@ class WXDLLIMPEXP_CORE wxGrid : public wxScrolledCanvas
     const wxColour *m_dragLastColour;
 
     // Row or column (depending on m_cursorMode value) currently being resized
+    // or -2 if row or col label is being resized
```

Comment:
We really ought to have a symbolic constant for this instead of repeating this `-2` many times.

But, to be honest, I rather dislike having such multiple-state variable. Having an int which can be invalid is not too bad, but having an int with 3 different kinds of values is not great. Could things become clearer if we introduced a separate `bool m_dragLabel`?